bin/launch-process

```diff
@@ -34,12 +34,15 @@ unless you know what you are doing.
     -h                Print this help text to stdout
 "
 
-# returns 1 if "$1" contains "$2", 0 otherwise.
+# echos 1 if "$1" contains "$2", 0 otherwise.
```

Comment:
can you update this comment, and add a little bit more info as to why printf is required over echo?

Reply:
done